examples/custom_allocator/lib.rs

```diff
@@ -0,0 +1,180 @@
+//! # Custom Allocator
+//!
+//! This example demonstrates how to opt-out of the ink! provided global memory allocator.
```

Comment:
Lemme be the guy commenting about something on the order of 80 CPL 😬 .

Reply:
Can we configure `rustfmt` to catch this? I think it enforces code CPL to 100, not sure about the comments.

Reply:
We already enforce this to 90 characters, which is what it's at 😉 

https://github.com/paritytech/ink/blob/b8862a1147a8fb577c82304ed1dec9a8c45c386a/.rustfmt.toml#L1